src/GRA.Controllers/ChallengesController.cs

```diff
@@ -73,9 +80,19 @@ public class ChallengesController : Base.UserController
                 }
                 filter.CategoryIds = categoryIds;
             }
-            if (Favorites && AuthUser.Identity.IsAuthenticated)
+            if (AuthUser.Identity.IsAuthenticated)
             {
-                filter.Favorites = true;
+                filter.Favorites = Favorites;
+                if (string.IsNullOrWhiteSpace(Status) 
```

Comment:
This should initialize to false so the only time it should need to be changed is if it should be true.

Reply:
Part of @iafb's changes

Reply:
IsCompleted defaults to null so a check to set to false is needed.